1_antenatal_care.do

```diff
@@ -109,6 +109,9 @@
 			if inlist(country_name,"CostaRica2018") {	
 				global mn3 "mn3a mn3b mn3i"
 			}
+			if inlist(country,name,"Tonga2019"){
+				global mn3 "mn3a mn3b mn3d mn3g"
```

Comment:
ANC p125: Medical Doctor, Nurse/Midwife, Health Officer and Community Nurse

Reply:
checked;

mn3g - Prenatal care provider: Community health worker
removed